api/maven-api-model/src/main/mdo/model.vm

```diff
@@ -85,7 +85,7 @@
       #end
     #end
 // =================== DO NOT EDIT THIS FILE ====================
-//   Generated by Maven, any modifications will be overwritten.
+//   ${generatedBy}
```

Comment:
Is this defined somewhere ? I can't find any definition for this variable.

Reply:
This I would like to know as well

Reply:
Here's the answer: https://github.com/codehaus-plexus/modello/blob/master/modello-plugins/modello-plugin-velocity/src/main/java/org/codehaus/modello/plugin/velocity/VelocityGenerator.java#L85

Reply:
even better: the user documenattion https://codehaus-plexus.github.io/modello/modello-plugins/modello-plugin-velocity/

Reply:
From this documentation point of view this makes sense... otherwise it is confusing because only `templatePath` should be a Velocity property.